src/include/distributed/worker_transaction.h

```diff
@@ -43,7 +43,12 @@ typedef enum TargetWorkerSet
 	 * All the active primary nodes in the metadata which have metadata
 	 * (includes the coodinator if it is added)
 	 */
-	METADATA_NODES
+	METADATA_NODES,
+
+	/*
+	 * All the active primary nodes in the metadata except the current node
+	 */
+	OTHER_NODES
```

Comment:
```suggestion
	/*
	 * All the active primary nodes in the metadata except the local node
	 */
	OTHER_NODES
```

Reply:
We can discuss but to me the word local means something in this node. So local node doesn't sound right.

Reply:
I don't think this needs to be introduced anymore. We already have REMOTE_NODES which is the same afaict. It was introduced by @onurctirtir in #7278. Let's also make all other places where we use the term "other nodes" to saying "remote nodes". e.g. the `execute_command_on_other_nodes` UDF.